lib/hanami/cli/commands/monolith/console.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "hanami/console/context"
+
+require_relative "../application"
+
+module Hanami
+  module CLI
+    module Commands
+      module Monolith
+        class Console < Application
+          desc "Application REPL"
+
+          def call(**_opts)
+            require "pry"
+
+            prompt = application_prompt
+
+            Pry.config.prompt = Pry::Prompt.new(
```

Comment:
@solnic why `pry` instead of IRB, which is part of Ruby distribution?

Reply:
@jodosha I haven't figured out how to set its prompt 😆 I'll give this another shot

Reply:
@solnic https://docs.ruby-lang.org/en/2.2.0/IRB.html#module-IRB-label-Customizing+the+IRB+Prompt

Reply:
@jodosha I spent almost two hours trying to figure out how to start IRB and gave up :(